src/js/tabs/ports.js

```diff
@@ -5,95 +5,67 @@ TABS.ports = {
 };
 
 TABS.ports.initialize = function (callback, scrollPosition) {
-    var self = this;
-
-    var board_definition = {};
-
-    var functionRules = [
-         {name: 'MSP',                  groups: ['configuration', 'msp'], maxPorts: 2},
-         {name: 'GPS',                  groups: ['sensors'], maxPorts: 1},
-         {name: 'TELEMETRY_FRSKY',      groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1},
-         {name: 'TELEMETRY_HOTT',       groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1},
-         {name: 'TELEMETRY_SMARTPORT',  groups: ['telemetry'], maxPorts: 1},
-         {name: 'RX_SERIAL',            groups: ['rx'], maxPorts: 1},
-         {name: 'BLACKBOX',     groups: ['peripherals'], sharableWith: ['msp'], notSharableWith: ['telemetry'], maxPorts: 1}
+    const self = this;
+
+    let board_definition = {};
+
+    const functionRules = [
+        { name: 'MSP',                  groups: ['configuration', 'msp'], maxPorts: 2 },
+        { name: 'GPS',                  groups: ['sensors'], maxPorts: 1 },
+        { name: 'TELEMETRY_FRSKY',      groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1 },
+        { name: 'TELEMETRY_HOTT',       groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1 },
+        { name: 'TELEMETRY_SMARTPORT',  groups: ['telemetry'], maxPorts: 1 },
+        { name: 'RX_SERIAL',            groups: ['rx'], maxPorts: 1 },
+        { name: 'BLACKBOX',     groups: ['peripherals'], sharableWith: ['msp'], notSharableWith: ['telemetry'], maxPorts: 1 },
+        { name: 'TELEMETRY_LTM',        groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1 },
+        { name: 'TELEMETRY_MAVLINK',    groups: ['telemetry'], sharableWith: ['msp'], notSharableWith: ['peripherals'], maxPorts: 1 },
+        { name: 'ESC_SENSOR', groups: ['sensors'], maxPorts: 1 },
+        { name: 'TBS_SMARTAUDIO', groups: ['peripherals'], maxPorts: 1 },
+        { name: 'IRC_TRAMP', groups: ['peripherals'], maxPorts: 1 },
+        { name: 'TELEMETRY_IBUS', groups: ['telemetry'], maxPorts: 1 },
+        { name: 'RUNCAM_DEVICE_CONTROL', groups: ['peripherals'], maxPorts: 1 },
+        { name: 'LIDAR_TF', groups: ['peripherals'], maxPorts: 1 },
     ];
 
-    if (semver.gte(FC.CONFIG.apiVersion, "1.15.0")) {
```

Comment:
Why have all of these checks been removed - this is incorrect now.

Reply:
We talked about in on slack, and it was safe to assume we should support 1.40 and higher, so I thought you would be happy with the removal.